daft/catalog/__init__.py

```diff
@@ -83,9 +83,14 @@ def unregister_catalog(catalog_name: str | None) -> bool:
         bool: True if a catalog was successfully unregistered, False otherwise.
 
     Example:
-        >>> import daft
-        >>> daft.unregister_catalog("my_catalog")
+        ``` py linenums="1"
+        import daft
+
+        daft.unregister_catalog("my_catalog")
+        ```
+        ```
         True
+        ```
```

Comment:
so these weren't RST, but were standard [doctest](https://docs.python.org/3/library/doctest.html) style docstrings. 

Without the `>>>` syntax, they won't run via `make doctests` anymore.